CONTRIBUTING.md

```diff
@@ -12,20 +12,66 @@ Python 3.9 and above are supported by the SDK.
 
 ### Installation and Dependencies
 
-We use [Hatch](https://hatch.pypa.io/) to manage the project.
+We use [uv](https://github.com/astral-sh/uv) for fast Python package management and dependency resolution.
 
-To install Hatch, just run `pip install hatch`.
+To install uv, follow the [installation guide](https://docs.astral.sh/uv/getting-started/installation/).
 
 You will also need to set up the `pre-commit` hooks.
 Run `pre-commit install` in the root directory of the repository.
-If you don't have `pre-commit` installed, you can install it with `pip install pre-commit`.
+If you don't have `pre-commit` installed, you can install it with `uv run pip install pre-commit`.
+
+### Setup Development Environment
+
+1. **Clone the repository:**
+   ```bash
+   git clone https://github.com/open-feature/python-sdk.git
+   cd python-sdk
+   ```
+
+2. **Install dependencies:**
+   ```bash
+   uv sync --extra dev
+   ```
 
 ### Testing
 
-Run tests with `hatch run test`.
+Run tests with `uv run pytest tests`.
 
 We use `pytest` for our unit testing, making use of `parametrized` to inject cases at scale.
 
+### Coverage
+
+Run coverage tests with:
+```bash
+uv run coverage run -m pytest tests/
+  # Generate coverage report
+```
+
+### End-to-End Tests
+
+Run e2e tests with behave:
+```bash
+# Update git submodules and run behave tests
+git submodule update --init --recursive
+cp spec/specification/assets/gherkin/* tests/features/
+uv run behave tests/features/
```

Comment:
can we maybe configure this in th pyproject.toml, so we only need to execute `uv run behave`

Reply:
Yeah, I've been looking for this kind of functionality.
Currently it doesn't look like there's a decent workaround.
See this thread: https://github.com/astral-sh/uv/issues/5903

They're adding support, but the issue has been there for almost a year.

Reply:
Using `project.scripts` is the closest thing I can think of without adding dependencies or other tools.
It'd really be an interim solution...